docker-compose.yml

```
@@ -15,7 +15,10 @@ services:
     build:
       context: .
       dockerfile: IdentityServer/Dockerfile.dev
+    depends_on:
+      - db
     environment: 
+      - ConnectionStrings__DefaultConnection=Server=172.16.238.2;Database=master;User=sa;Password=Dexcelence!1;
```

Small note: It is better to separate the IDSRV db from the application db. If you don't want to do that in a test environment then i would suggest using different dbs inside the mssql docker. A discussion about this can be found [here](https://stackoverflow.com/questions/39204142/docker-compose-with-multiple-databases).

---

```suggestion
      - ConnectionStrings__DefaultConnection=Server=172.16.238.2;Database=identity;User=sa;Password=Dexcelence!1;
```

Thanks for the suggestion, I think the above-suggested code should do the job. For development I don't think we need a different solution, do you agree?